src/groovy/org/pih/warehouse/core/ReasonCode.groovy

```diff
@@ -64,6 +64,8 @@ enum ReasonCode {
     REJECTED(32),
     REQUEST_ERROR(33),
     NOT_NEEDED_BASED_ON_CONSUMPTION(34),
+    CLINICAL_JUDGMENT(35),
+    INSUFFICIENT_QUANTITY_AVAILABLE(36),
```

Comment:
These aren't actually reason codes. They are classifications for all of the reason codes. This will be fine for now, but we need to convert these to be in a separate enum or perhaps related as parents to each of the ReasonCode enum values.